ehcache-core/src/main/java/org/ehcache/core/spi/store/tiering/CachingTier.java

```diff
@@ -107,6 +108,22 @@ public interface CachingTier<K, V> extends ConfigurationChangeSupport {
    */
   void setInvalidationListener(InvalidationListener<K, V> invalidationListener);
 
+  /**
+   * Bulk method to get its mapped value from CachingTier, then look at AuthoritativeTier for the missing entries Of CachingTier then Compute a value for every key passed in the {@link Set} <code>keys</code> argument using the <code>mappingFunction</code>
+   * to compute the value.
+   * <p>
+   * The function gets an {@link Iterable} of {@link java.util.Map.Entry} key/value pairs, where each entry's value is its currently stored value
+   * for each key that is not mapped in the store. It is expected that the function returns an {@link Iterable} of {@link java.util.Map.Entry}
+   * key/value pairs containing an entry for each key that was passed to it. This returned {@link Iterable} should also iterate in the same order as the input {@link Iterable}.
```

Comment:
Again, we can strike the sentence about iteration order.

Reply:
removed it